lib/Params.mli

```diff
@@ -36,6 +37,15 @@ module Exp : sig
     -> Fmt.t
 end
 
+module Mod : sig
+  type args =
+    { dock: bool  (** Whether each argument's [pro] should be docked. *)
+    ; arg_psp: Fmt.t  (** Break before every arguments. *)
```

Comment:
What do you think of using a `Fmt.sp` here?

Reply:
That was my initial idea but it's `str " "` in some cases. I come to the conclusion that `Fmt.t` is generally better than `Fmt.sp`.

Reply:
You can use `Fmt.Blank` for `" "`